### PR TITLE
Make fromname required, reorder request vars

### PR DESCRIPTION
**Pull request by ros3bud**
solves issues 13 & 14

- add fromname to params sent to aws
- updated tests to reflect change
- make fromname required
- re-record nocks to reflect the updates

### CODE REVIEW

**Comment by ros3bud** [comment]
@cgrayson, LGTY?

File: spec/send_spec.coffee [hunk 1]
```diff
@@ -109,28 +116,28 @@ describe 'successfully send email', ->
...
-      )
+      .post('/', "Action=SendEmail&Destination.BccAddresses.member.1=cheese%40eat.com&Destination.ToAddresses.member.1=lynda%40activeprospect.com&Message.Body.Html.Data=%3Cp%3E%20Best%20new%20recipes%20%3C%2Fp%3E&Message.Body.Text.Data=Best%20new%20chefs&Message.Subject.Data=Food%20and%20Wine&Source=%27Test%20Tester%27%20%3Cno-reply%40offerconfirmation.com%3E&Version=2010-12-01")
+      .reply(200, "<SendEmailResponse xmlns=\"http://ses.amazonaws.com/doc/2010-12-01/\">\n  <SendEmailResult>\n    <MessageId>0000014ef464e130-3d622266-67e1-4116-89cd-06a78dd229c7-000000</MessageId>\n  </SendEmailResult>\n  <ResponseMetadata>\n    <RequestId>b815b3dd-39fc-11e5-90b9-2f1f70eb5c9b</RequestId>\n  </ResponseMetadata>\n</SendEmailResponse>\n", 
+          { 
+            'x-amzn-requestid': 'b815b3dd-39fc-11e5-90b9-2f1f70eb5c9b'
+            'content-type': 'text/xml'
+            'content-length': '326'
+            'date': 'Mon, 03 Aug 2015 16:28:48 GMT' 
+          }
+        )
```
Comment (cgrayson) [inline spec/send_spec.coffee]: Why extra spaces here?

**Comment by cgrayson** [comment]
Other than that minor question, LGTM

File: spec/send_spec.coffee (hunk 1 above)
Comment (ros3bud) [inline spec/send_spec.coffee]: Good catch. Fixed.

**Pull request merged**